Components/Maps/Maps.js

```diff
@@ -35,11 +67,45 @@ export const Maps = ({ route }) => {
       coordinate={marker.coordinates}
       title={marker.title}
       description={marker.description}
-      onPress={(marker) => addActivity(marker.nativeEvent.coordinate)}
+      onPress={(marker) => handleActivity(marker.nativeEvent.coordinate)}
       />
     )
   })
-    
+
+  // addActivity(dest.id, name, date, address, category, rating, image, lat, long)
+    // setSavedActivities([...savedActivities, clickedActivity])
+
+
+  const handleYesClick = async () => {
+    await addActivity(dest.id, clickedActivity.title, endDate, clickedActivity.address, clickedActivity.description, clickedActivity.rating, clickedActivity.image, clickedActivity.coordinates.latitude, clickedActivity.coordinates.longitude)
+      .then(data => {
+        setSavedActivities([...savedActivities, data.data.createActivity.activity])
+        setClickedActivity('')
+        handleTripsFetch()
+      })
+      .catch(err => console.log(err))
+  }
```

Comment:
Good work on re-rendering the agenda upon posting the saved activities. Looks nice all on one page